core/controllers/base.py

```diff
@@ -295,8 +299,78 @@ def dispatch(self):
                 self.handle_exception(e, self.app.debug)
                 return
 
+        handlers_has_passed_schema_validation = True
```

Comment:
You don't need this, it just adds complexity. Just return in the various branches.

Reply:
The reason behind I have used an extra variable is that in the dispatch function there are already 5 return statements, if I add 2 more then it will cross max return statements i.e, 6.
The error I faced is,
“R:244, 4: Too many return statements (7/6) “
Please suggest any better way, looking forward to your suggestions.

Reply:
Oh I see -- OK, fine to leave it as-is, thanks for explaining. I suggest changing the name though, perhaps to `schema_validation_succeeded` (it's not the handlers which are being tested, it's the request).

Reply:
Done